src/OpenTelemetry.Abstractions/Trace/ISpanBuilder.cs

```diff
@@ -53,6 +62,17 @@ public interface ISpanBuilder
         /// <returns>This span builder for chaining.</returns>
         ISpanBuilder SetNoParent();
 
+        /// <summary>
+        /// Sets flag indicating that new span should become a child of implicit context (Activity.Current)
+        /// or continue run in this context and inherit it
+        /// Use this method with value 'false' in auto-collectors when library is instrumented with Activities.
+        /// Any parent that was set previously will be discarded.
+        /// </summary>
+        /// <param name="createChild">If true, a new span will become a child of the existing implicit context.
+        /// If false, new span will continue this context.</param>
+        /// <returns>This span builder for chaining.</returns>
+        ISpanBuilder SetCreateChild(bool createChild);
```

Comment:
this name is quite confusing. Why not move this part of deciding on whether to create one out of `Activity.Current` or a new one into the `Tracer`? This decision is only actual when you creating a SpanBuilder initially, right?